src/test/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImplTest.java

```diff
@@ -63,6 +101,39 @@ public void tearDown() throws Exception {
         cuExtractPaymentServiceImpl = null;
         FileUtils.forceDelete(new File(ACH_BATCH_TEST_FILE_PATH).getAbsoluteFile());
     }
+    
+    @Test
+    public void testDvCheckStubCreation() {
+        int line1ArrayPosition = 0;
+        int line2ArrayPosition = 1;
+        int line3ArrayPosition = 2;
+        
+        for (int numTests = 1; numTests <= MAX_NUM_DV_CHECK_STUB_NOTES_TESTS; numTests++) {
+            String stubLine1 = EDOC_DV_IDENTIFIER_AND_NUMBER;
+            
+            String stubLine2 = cuExtractPaymentServiceImpl.obtainTruncatedNoteLineSection(DV_NOTES_TEST_DATA[line1ArrayPosition], stubLine1);
+            stubLine1 = stubLine1 + cuExtractPaymentServiceImpl.obtainLeadingNoteLineSection(DV_NOTES_TEST_DATA[line1ArrayPosition], stubLine1);
```

Comment:
The unit test explicitly combines the old stub text and the potentially-truncated line text, whereas the equivalent code in CuExtractPaymentServiceImpl does not. Is this only for comparison purposes, or is the related CuExtractPaymentServiceImpl code supposed to behave that way as well?

Reply:
Unit test and code should be the same. Modified both to use String.concat.